pkg/factory/risingwave_object_factory.go

```diff
@@ -323,23 +327,11 @@ func (f *RisingWaveObjectFactory) envsForEtcd() []corev1.EnvVar {
 	}
 }
 
-func (f *RisingWaveObjectFactory) envsForMetaArgs() []corev1.EnvVar {
+func (f *RisingWaveObjectFactory) coreEnvsForMeta() []corev1.EnvVar {
```

Comment:
What is difference between `coreEnvsForMeta` vs `envsForMetaArgs` I see that `coreEnvsForMeta` is newly introduced.

Reply:
`coreEnvsForMeta` contains the envs that both standalone and clustered meta node require, such as for state stores and meta stores. It excludes all other options including those to specify the listening ports because the two modes differs on the way to set them.